homeassistant/components/zwave_js/services.py

```diff
@@ -75,15 +75,15 @@ async def async_set_config_parameter(self, service: ServiceCall) -> None:
         """Set a config value on a node."""
         nodes: Set[ZwaveNode] = set()
```

Comment:
Are you sure that ZWaveNode is hashable ?
I can't remember we have a __hash__ method in the model

Reply:
tests are passing, and I threw in a logging statement and failed the test purposely to check:
`ERROR    homeassistant.components.zwave_js.services:services.py:87 {Node(node_id=52)}`

Reply:
I think the [representation](https://github.com/home-assistant-libs/zwave-js-server-python/blob/master/zwave_js_server/model/node.py#L86) is what makes it hashable. Strictly taken the homeid should be part of the hash but this is good enough

Reply:
We can improve that in the library for the next release

Reply:
Custom classes uses the built in function `id` to get a default hash.
https://docs.python.org/3/glossary.html#term-hashable
https://docs.python.org/3/library/functions.html#id

Reply:
Ah, that is good enough in this case as we're iterating the ZWaveNode instances here